docs/weaviate/manage-collections/collection-aliases.mdx

```diff
@@ -0,0 +1,330 @@
+
+:::caution Technical preview
+
+Collection aliases were added in **`v1.32`** as a **technical preview**.<br/><br/>
+This means that the feature is still under development and may change in future releases, including potential breaking changes.
+**We do not recommend using this feature in production environments at this time.**
+
+:::
+
+Collection aliases allow you to create alternative names for your collections. This is useful for seamless schema migrations, A/B testing, or providing more convenient names for collections. An alias acts as a pointer to a collection - when you query using an alias name, Weaviate automatically routes the request to the target collection.
+
+## Create an alias
+
+To create an alias, specify the alias name and the target collection it should point to.
+
+<Tabs groupId="languages">
+  <TabItem value="py" label="Python Client v4">
+    <FilteredTextBlock
+      text={PyCode}
+      startMarker="# START CreateAlias"
+      endMarker="# END CreateAlias"
+      language="py"
+    />
+  </TabItem>
+
+<TabItem value="js" label="JS/TS Client v3">
+
+```typescript
+// JS/TS support coming soon
+```
+
+</TabItem>
+
+<TabItem value="java" label="Java">
+
+```java
+// Java support coming soon
+```
+
+</TabItem>
+
+  <TabItem value="go" label="Go">
+
+```go
+// Go support coming soon
+```
+
+  </TabItem>
+</Tabs>
+
+:::note
+
+- An alias name must be unique and cannot match any existing collection or alias name
+- Multiple aliases can point to the same collection
+- Aliases can be used anywhere a collection name is expected
+  :::
+
+## List all aliases
+
+Retrieve all aliases in your Weaviate instance.
+
+<Tabs groupId="languages">
+  <TabItem value="py" label="Python Client v4">
+    <FilteredTextBlock
+      text={PyCode}
+      startMarker="# START ListAllAliases"
+      endMarker="# END ListAllAliases"
+      language="py"
+    />
+  </TabItem>
+
+<TabItem value="js" label="JS/TS Client v3">
+
+```typescript
+// JS/TS support coming soon
+```
+
+</TabItem>
+
+<TabItem value="java" label="Java">
+
+```java
+// Java support coming soon
+```
+
+</TabItem>
+
+  <TabItem value="go" label="Go">
+
+```go
+// Go support coming soon
+```
+
+  </TabItem>
+</Tabs>
+
+## List aliases for a specific collection
+
+Get all aliases that point to a specific collection.
+
+<Tabs groupId="languages">
+  <TabItem value="py" label="Python Client v4">
+    <FilteredTextBlock
+      text={PyCode}
+      startMarker="# START ListCollectionAliases"
+      endMarker="# END ListCollectionAliases"
+      language="py"
+    />
+  </TabItem>
+
+<TabItem value="js" label="JS/TS Client v3">
+
+```typescript
+// JS/TS support coming soon
+```
+
+</TabItem>
+
+<TabItem value="java" label="Java">
+
+```java
+// Java support coming soon
+```
+
+</TabItem>
+
+  <TabItem value="go" label="Go">
+
+```go
+// Go support coming soon
+```
+
+  </TabItem>
+</Tabs>
+
+## Get a specific alias
```

Comment:
wdyt about `Get alias details`?

Imo `get` is ambiguous as it could be get to use